nuxeo-distribution/nuxeo-server-hotreload-tests/src/test/java/org/nuxeo/ftest/server/hotreload/ITDevHotReloadTest.java

```diff
@@ -50,6 +52,15 @@
  */
 public class ITDevHotReloadTest {
 
+    /** @since 11.1 **/
+    public static final String MIME_TYPE = "mime-type";
+
+    /** @since 11.1 **/
+    public static final String CUSTOM_IMAGE_PATH = "custom:image";
+
+    /** @since 11.1 **/
+    public static final String CUSTOM_QUESTION_PATH = "custom:question";
```

Comment:
I'm not in favor of having constant which are one test related because it could become a mess. I think locale variable should be enough.

Reply:
i see, i introduce them to avoid duplication code, but i think to be consistent with the rest of the test code i will remove them :)